lib/gems/pending/util/miq-password.rb

```diff
@@ -7,6 +7,7 @@ class MiqPasswordError < StandardError; end
 
   CURRENT_VERSION = "2"
   REGEXP = /v([0-9]+):\{([^}]*)\}/
+  REGEXP_V2_ENCODED = /v([0-9]+)%3A%7B(.*?)%7D/ # for URL encoded string of "v2:{...}"
```

Comment:
I think the `V2` should be removed from the name since the regex supports `v*`

Reply:
Why `(.*?)`?

Reply:
Also, we probably don't need the capture groups since we would have to URI decode the string before we could do anything with those values